src/graph/validator/ACLValidator.cpp

```diff
@@ -131,6 +131,21 @@ Status RevokeRoleValidator::toPlan() {
                                        sentence->getAclItemClause()->getRoleType());
 }
 
+// describe user
+Status DescribeUserValidator::validateImpl() {
+  auto sentence = static_cast<DescribeUserSentence *>(sentence_);
+  if (sentence->account()->size() > kUsernameMaxLength) {
```

Comment:
You should check whether the current user has the permission to describe the `account`.